embossc

```diff
@@ -46,6 +46,10 @@ def _parse_args(argv):
                       nargs=1,
                       default=".",
                       help="Prefix to use for the generated output file.")
+  parser.add_argument("--output-file",
+                      nargs=1,
+                      help="""Exact file path to use for the generated output
+                              file. Overrides --output-path is ignored.""")
```

Comment:
> Overrides --output-path is ignored.

"Only one of --output-path and --output-file may be used", maybe?  And update the --output-path message and throw an error if both are specified.

Reply:
That works for me.  And you are correct, there is inconsistent dash/underscore used here which is beyond the scope of this PR, so disregard that comment.

Reply:
I reworked closer to what Aaron suggested since it seems more straightforward relative to existing behavior and  flags. Now  --output-file help is "File name to be used for the generated output file. Defaults to input_file suffixed by '.h'". PTAL